index.js

```diff
@@ -21,18 +22,22 @@ module.exports.toTorrentFile = encodeTorrentFile
  * @return {Object}
  */
 function parseTorrent (torrentId) {
+  if (torrentId instanceof ArrayBuffer || ArrayBuffer.isView(torrentId)) {
+    torrentId = new Uint8Array(torrentId)
```

Comment:
If `torrentId` is already a `Uint8Array`, this will copy it into a new `Uint8Array` unnecessarily.